docs/versioned_docs/version-2.11.0/troubleshooting/t-cstor.md

```diff
@@ -0,0 +1,513 @@
+
+**Troubleshooting**
+To bring cStor pool back to online state carry out the below mentioned steps,
+
+1. **Update validatingwebhookconfiguration resource's failurePolicy**:
+   Update the `validatingwebhookconfiguration` resource's failure policy to `Ignore`. It would be previously set to `Fail`. This informs the kube-APIServer to ignore the error in case cStor admission server is not reachable.
+   To edit, execute:
+
+   ```
+   $ kubectl edit validatingwebhookconfiguration openebs-cstor-validation-webhook
+   ```
+
+   Sample Output with updated `failurePolicy`
+
+   ```
+    kind: ValidatingWebhookConfiguration
+    metadata:
+      name: openebs-cstor-validation-webhook
+      ...
+      ...
+    webhooks:
+    - admissionReviewVersions:
+      - v1beta1
+    failurePolicy: Fail
+      name: admission-webhook.cstor.openebs.io
+    ...
+    ...
+
+   ```
+
+2. **Scale down the admission**:
+
+   The openEBS admission server needs to be scaled down as this would skip the validations performed by cStor admission server when CSPC spec is updated with new node details.
+
+   ```
+   $ kubectl scale deploy openebs-cstor-admission-server -n openebs --replicas=0
+   ```
+
+   Sample Output:
+
+   ```
+   deployment.extensions/openebs-cstor-admission-server scaled
+   ```
+
+3. **Update the CSPC spec nodeSelector**:
+   The `CStorPoolCluster` needs to be updated with the new `nodeSelector` values. The updated CSPC now points to the new nodes instead of the old nodeSelectors.
+
+   Update `kubernetes.io/hostname` with the new values.
+
+   Sample Output:
+
+   ```shell hideCopy
+   apiVersion: cstor.openebs.io/v1
+   kind: CStorPoolCluster
+   metadata:
+   name: cstor-cspc
+   namespace: openebs
+   spec:
+   pools:
+       - nodeSelector:
+           kubernetes.io/hostname: "ip-192-168-25-235"
+       dataRaidGroups:
+       - blockDevices:
+           - blockDeviceName: "blockdevice-798dbaf214f355ada15d097d87da248c"
+       poolConfig:
+           dataRaidGroupType: "stripe"
+       - nodeSelector:
+           kubernetes.io/hostname: "ip-192-168-33-15"
+       dataRaidGroups:
+       - blockDevices:
+           - blockDeviceName: "blockdevice-4505d9d5f045b05995a5654b5493f8e0"
+       poolConfig:
+           dataRaidGroupType: "stripe"
+       - nodeSelector:
+           kubernetes.io/hostname: "ip-192-168-75-156"
+       dataRaidGroups:
+       - blockDevices:
+           - blockDeviceName: "blockdevice-c783e51a80bc51065402e5473c52d185"
+       poolConfig:
+           dataRaidGroupType: "stripe"
+   ```
+
+To apply the above configuration, execute:
+
+```
+$ kubectl apply -f cspc.yaml
+```
```

Comment:
add space here, so that it will come inside the li

Reply:
used it as list number 4, although it is not defined so (https://docs.openebs.io/docs/next/t-cstor.html)